consensus/polybft/bridge.go

```diff
@@ -74,10 +74,20 @@ func (b bridge) PostBlock(req *PostBlockRequest) error {
 // PostEpoch is a function executed on epoch ending / start of new epoch
 // and calls PostEpoch in each bridge manager
 func (b bridge) PostEpoch(req *PostEpochRequest) error {
+	isInserted := false
+
 	for chainID, bridgeManager := range b {
 		if err := bridgeManager.PostEpoch(req); err != nil {
 			return fmt.Errorf("erorr bridge post epoch, chainID: %d, err: %w", chainID, err)
 		}
+
+		if !isInserted {
```

Comment:
This is not what I had in mind. Just call the state directly here to insert epoch for internal chain, and then run a for loop and call InsertEpoch on bridge event managers.

You dont need insert internal/external functions. Just make sure that InsertEpoch on bridge event managers only inserts epoch to state for their external chain id.